glyph/src/glyph.rs

```diff
@@ -76,6 +76,17 @@ pub struct GlyphImage<'a> {
     pub format: GlyphImageFormat,
 }
 
+#[derive(Debug, Clone)]
+pub struct GlyphSvg<'a> {
```

Comment:
We can make it non-exhaustive to allow us to add new fields later (e.g. perhaps in a breaking ttf-parser change) without breaking semver

```suggestion
#[non_exhaustive]
pub struct GlyphSvg<'a> {
```

Reply:
sure!